source/MaterialXRenderOsl/OslRenderer.cpp

```diff
@@ -170,35 +165,41 @@ void OslRenderer::renderOSL(const FilePath& dirPath, const string& shaderName, c
     }
     command += " > " + errorFile + redirectString;
 
-    int returnValue = std::system(command.c_str());
-
-    std::ifstream errorStream(errorFile);
-    StringVec result;
-    string line;
-    const string pngWarning("libpng warning: iCCP: known incorrect sRGB profile");
```

Comment:
Does that mean that oiio error does not come up when running the test suite ?
Might want to keep ignoring this in case it shows up when folks are using a different version of oiio ?

Reply:
I haven't encountered this error in the test suite so far, though you could be right that it may persist with some versions of OpenImageIO.  My recommendation would be to start with this approach, and then add back error suppression based on what we encounter in the latest codebase.